package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.0.1",
   "description": "",
   "main": "index.js",
+  "type": "module",
   "directories": {
     "example": "examples"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc",
+    "test": "jest",
+    "test-c": "jest --coverage"
```

Comment:
Running `test` command end up with error:
```
ReferenceError: module is not defined in ES module scope ...
This file is being treated as an ES module because it has a '.js'  ...
```
![image](https://user-images.githubusercontent.com/442111/225523333-641a3f93-4a93-42af-916e-b2f7dc29e939.png)

Reply:
Rename extensions of `jest.config.js` to `.cjs` makes it work.

Reply:
However tests on my machine failing:
![image](https://user-images.githubusercontent.com/442111/225523983-951fc456-2b8c-4272-ac92-a8a5bcd2c75d.png)

Reply:
This problem is solved. 

change 
``` typescript
    const couchdbAdapter = await CouchdbAdapter.newAdapter(
      "http://admin:123456@localhost:5984"
    );
```
to 
``` typescript
    const couchdbAdapter = await CouchdbAdapter.newAdapter(
      "http://admin:123456@127.0.0.1:5984"
    );
```

make it work.

<img width="774" alt="8e21e66698af77099bcfa454987ef1d" src="https://user-images.githubusercontent.com/84124955/225955591-e5a7ab21-37ca-4aba-a3df-b8e7be3d02da.png">